pkg/container/types/datetime.go

```diff
@@ -215,6 +215,13 @@ func FromUnix(time int64) Datetime {
 	return Datetime((time + unixEpoch) << 20)
 }
 
+func UnixToTimestamp(time int64) Timestamp {
+	localTZAligned := localTZ << 20
+	dt := FromUnix(time)
+	dtint64 := *(*int64)(unsafe.Pointer(&dt))
```

Comment:
What is this cast?   Really?     You need to have a Datetime.toInt64 method and it cannot have anything to do with unsafe.

Reply:
OK, that's a good idea. I'll adopt the way you said